# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintJob.cpp
# Commit: 853cf247e870
# Full Hash: 853cf247e8703644e30a20956eaf14b091efa6f9
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p8 - Centralize nulling out of nsPrintJob::mPrt. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146981
# ==============================================================================

diff -r 2448004443d5 -r 853cf247e870 layout/printing/nsPrintJob.cpp
--- a/layout/printing/nsPrintJob.cpp	Sun May 22 09:37:57 2022 +0000
+++ b/layout/printing/nsPrintJob.cpp	Sun May 22 09:37:58 2022 +0000
@@ -239,7 +239,7 @@
   }
   mIsDestroying = true;
 
-  mPrt = nullptr;
+  DestroyPrintingData();
 
   mDocViewerPrint = nullptr;
 }
@@ -358,7 +358,7 @@
     if (rv != NS_ERROR_ABORT && rv != NS_ERROR_OUT_OF_MEMORY) {
       FirePrintingErrorEvent(rv);
     }
-    mPrt = nullptr;
+    DestroyPrintingData();
   }
 
   return rv;
